packages/shared/src/lib/core/profile/migrations/prod/prod-profile-migration-5-to-6.ts

```diff
@@ -1,8 +1,10 @@
+import { AppStage } from '@core/app/enums'
 import { IPersistedProfile } from '@core/profile/interfaces'
 
 export function prodProfileMigration5To6(existingProfile: unknown): Promise<void> {
     const profile = existingProfile as IPersistedProfile
     // @ts-expect-error isDeveloperProfile is removed in favor of the feature flag
     delete profile.isDeveloperProfile
+    profile.versionTrack = AppStage.ALPHA
```

Comment:
Shouldnt this be `AppStage.PROD`?

Reply:
+ on migrating profiles, should be check this flag to determine which migration pipeline we're going to call for this profile?

Reply:
Updated to PROD.

Ideally yes we should check the track, but because we need profiles to be migrated to this, it may clash because we don't have it yet